bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java

```diff
@@ -83,6 +108,21 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
             role = roleMappings.get(splitArgs[1].toLowerCase());
         }
 
+        if (pr.author().equals(command.user()) && !censusInstance.isReviewer(command.user())) {
```

Comment:
Maybe simplify this to just `! isReviewer`? Including the test for `is author` doesn't really add value (removing it seems to better reflect the intent and might be safer / more future proof).

Reply:
No, we don't want people who are _not_ the PR author and who are _not_ Reviewers to be able to increase the required number of reviewers. Allowing that makes it possible to harass pull request by e.g. registering a spam account and just drop a bunch `/reviewers 9` in a bunch of PRs.

Reply:
I agree, but that isn't related to this block (your concern is a good one and is handled by the previous block). By adding the `author &&` to `!reviewer` it makes this less restrictive not more. In any case, it doesn't matter...at least not given the prior test.